php/php.dbgp/src/org/netbeans/modules/php/dbgp/packets/Property.java

```diff
@@ -64,12 +66,10 @@ public String getName() {
     }
 
     public void setName(String value) {
-        Node node = getNode().getAttributes().getNamedItem(NAME);
-        if (node == null) {
-            node = getNode().getOwnerDocument().createAttribute(NAME);
-            getNode().appendChild(node);
+        if (getNode() instanceof Element) {
```

Comment:
What if node is not an `Element`? Can it happen? Is NOOP in such a case expected? I don't know it, just asking :)

Reply:
If I `node` is not `Element` then also `getName()` will not work and it will return `null`.
So no effect in this case is OK.

Reply:
OK, thanks for the explanation.